README.md

```diff
@@ -1,3 +1,4 @@
 # Lavinia-api
 ## Prerequisites
+* [.NET Core Runtime 2.1.2](https://www.microsoft.com/net/download/thank-you/dotnet-runtime-2.1.2-windows-hosting-bundle-installer)
```

Comment:
It is not necessary to download the SDK for running the API -- say you are a front-end developer, the only thing you need on your computer to make this work is the correct runtime. It is convoluted to require the installation of an SDK bundle that may or may not change over time depending on Microsoft's wishes.

I have previously had issues determining which version should be used for a .NET Core project due to Microsoft's own versioning scheme (There are numerous versions of 2.0 that are called 2.1, etc, hence the specific download required).

I agree it might be valid to add the SDK as an alternative in the prerequisites though.